app/lib/bk/compat/parsers/gha.rb

```diff
@@ -2,7 +2,9 @@
 
 require_relative '../error'
 require_relative '../translator'
-require_relative '../pipeline'
+require_relative '../models/pipeline'
+require_relative '../models/steps/gha/gha_step'
```

Comment:
this file no longer uses any classes from this file so it should not have this line

Reply:
Adjusted 👍